smarty/templates/main.tpl

```diff
@@ -296,35 +296,5 @@
         {if $FormAction}
         </form>
         {/if}
-
```

Comment:
Hi @ridz1208, I think this extra html that was being added to Loris is the problem.

Reply:
A comment on #8309 refreshed my memory of how this is supposed to work and why it's not working. This "extra" HTML is used by .ajaxError to trigger a popup modal after a timeout where the user can log back in without losing data.

It's not working anymore because it was using `$(document).ajaxError` and we're using the fetch API instead of jQuery in most places now.

From what I can tell, there's no way to globally hook in to the fetch API to check for 403 errors like this was doing so it's probably safe to remove (https://stackoverflow.com/questions/43594266/fetch-equivalent-to-jquerys-document-ajaxerror) this HTML since we're not going back to jQuery's `$.ajax`.

Can you remove this JS code too which will now no longer work even for `$.ajax`? https://github.com/aces/Loris/blob/main/htdocs/js/helpHandler.js#L98-L123?